lib/hermetic-common/src/schema.js

```diff
@@ -156,6 +161,52 @@ const technologySchema = Joi.object({
+    .description('Very short generic name which describes the essential purpose of the system e.g.Student Management System, Research Information Management System'),
+  commsResponsibility: Joi.string().optional().max(100)
+    .description('Where does responsibility for communications concerning system and process change reside? e.g. ITS, department (which one)'),
+  integrationResponsibility: Joi.string().optional().max(100)
+    .description('Who develops the code for system integrations e.g.ITS, vendor, third party?'),
+  licensingResponsibility: Joi.string().optional().max(100)
+    .description('Where does responsibility and funding for licence renewals reside? E.g. ITS/central budget, departmental budget (which dept/role/person)'),
+  reportWriting: Joi.string().optional().max(100)
+    .description('Who is responsible for developing operational reporting from the system e.g. ITS, department (which one), vendor, users'),
+  userAccessControl: Joi.string().optional().max(100)
+    .description('Who is responsible for granting access setting up roles/role groups etc? e.g. department/job role (not a named person), ITS'),
+  trainingDelivery: Joi.string().optional().max(100)
+    .description('Who delivers training to end users e.g. ITS, vendor, third party?'),
+  vendorManagement: Joi.string().optional().max(100)
+    .description('Who is responsible for vendor relationship management e.g. regular meetings, point of contact for new feature descriptions, management of support contract etc. e.g. ITS, other dept?'),
+  applicationHelp: Joi.string().optional().max(100)
+    .description('Who provides users with help about how to use the application i.e. where is the users first point of contact for help in the application?'),
+  comments: Joi.string().optional().max(1024)
+    .description('General comments relevant to supporting the system'),
+  incidentsAndProblems: Joi.string().optional().max(100)
+    .description('Where are incident and problem management executed e.g. ITS, vendor, third party?'),
+  internalHosting: Joi.string().optional().valid('Yes', 'No')
+    .description('Does ITS host the application e.g. servers, backup, server patching'),
+  sME: Joi.string().optional().max(1024).label('SME')
+    .description('Who is considered a Subject Matter Expert - could be more than one person.'),
+  enhancements: Joi.string().optional().max(100)
+    .description('Who does the work in delivering application enhancements (build or configure) e.g.ITS, vendor, third party?'),
+  upgradeDecisions: Joi.string().optional().max(100)
+    .description('Who is responsible for decisions around upgrade dates, feature adoption etc. e.g. ITS, department/job role (not named person)'),
+  upgradeFundingSource: Joi.string().optional().max(100)
+    .description('How are major upgrades and/or enhancements funded e.g. ITS operational budget, ITS Group 1 capital funding (repeating), Group 2 capital funding (one off), Departmental budget, none defined'),
```

Comment:
This text doesn't seem appropriate for an open source project.  The other references to ITS are iffy, but this one is extremely business specific